client/ayon_core/hosts/maya/api/lib.py

```diff
@@ -2565,11 +2571,8 @@ def get_frame_range(include_animation_range=False):
         # Some usages of this function use the full dictionary to define
         # instance attributes for which we want to exclude the animation
         # keys. That is why these are excluded by default.
-        task_name = get_current_task_name()
+
         settings = get_project_settings(project_name)
-        task_entity = ayon_api.get_task_by_name(
-            project_name, folder_entity["id"], task_name
-        )
         task_type = None
         if task_entity:
             task_type = task_entity["taskType"]
```

Comment:
Looking at the the code above - where we now get attributes directly from the task entity I don't think there's ever a moment where `if task_entity` should be empty or `None` as such this if statement makes no sense anymore?
```suggestion
        task_type = task_entity["taskType"]
```